src/test/java/com/softserveinc/ita/rozetka_test/OrderTest.java

```diff
@@ -4,21 +4,44 @@
 import com.softserveinc.ita.rozetka.page_objects.BasketPage;
 import com.softserveinc.ita.rozetka.page_objects.HomePage;
 import com.softserveinc.ita.rozetka.page_objects.OrderPage;
+
+import org.testng.Assert;
 import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
 
 public class OrderTest extends TestRunner {
 
     //This added for future tests
     private OrderPage openOrderPage;
     private BasketPage openBasketPage;
+    private String searchString;
 
     //This added for future tests
     @BeforeMethod
-    public void addProductToBasketOpenOrder() {
+    public void putProductToBasketOpenOrder() {
         openOrderPage = new HomePage()
                 .openHomePage()
                 .openProductByNumber(1)
                 .addProductToBasket()
                 .openOrderPage();
     }
+
+    @Test
+    public void insertBuyersCredentialsTest() {
+        String surname = "Петренко";
+        String name = "Вася";
+        String city = "Львів";
+        String phone = "0997003330";
+        boolean isDisplayedConfirmOrderButton = openOrderPage
+                .setName(name)
+                .setSurname(surname)
+                .setCity(city)
+                .setPhone(phone)
+                .isDisplayedConfirmOrderButton();
```

Comment:
group these data into ShippingAddress class, add method setShippingAddress() to openOrderPage